src/main/java/net/hypixel/nerdbot/command/ItemGenCommands.java

```diff
@@ -126,83 +281,86 @@ public void generateItem(
         padding = Objects.requireNonNullElse(padding, 0);
         padding = Math.max(0, padding);
 
-        File minecraftImage = new MinecraftImage(colorParser.getParsedDescription(), MCColor.GRAY, 500, alpha, padding)
-            .render()
-            .toFile();
-
-        event.getHook().sendFiles(FileUpload.fromData(minecraftImage)).setEphemeral(false).queue();
+        MinecraftImage minecraftImage = new MinecraftImage(colorParser.getParsedDescription(), MCColor.GRAY, 500, alpha, padding).render();
 
         Member member = event.getMember();
         DiscordUser discordUser = Util.getOrAddUserToCache(NerdBotApp.getBot().getDatabase(), member.getId());
         if (discordUser == null) {
             log.info("Not updating last item generator activity date for " + member.getEffectiveName() + " (ID: " + member.getId() + ") since we cannot find a user!");
-            return;
+        } else {
+            discordUser.getLastActivity().setLastItemGenUsage(System.currentTimeMillis());
+            log.info("Updating last item generator activity date for " + member.getEffectiveName() + " to " + System.currentTimeMillis());
         }
 
-        discordUser.getLastActivity().setLastItemGenUsage(System.currentTimeMillis());
-        log.info("Updating last item generator activity date for " + member.getEffectiveName() + " to " + System.currentTimeMillis());
-    }
-
-    @AutocompletionHandler(name = "rarities", mode = AutocompletionMode.CONTINUITY, showUserInput = false)
-    public Queue<String> listRarities(CommandAutoCompleteInteractionEvent event) {
-        return Stream.of(Rarity.VALUES).map(Enum::name).collect(Collectors.toCollection(ArrayDeque::new));
+        return minecraftImage;
     }
 
-    @JDASlashCommand(name = "infogen", description = "Get a little bit of help with how to use the Generator bot.")
-    public void askForInfo(GuildSlashEvent event) {
-        String senderChannelId = event.getChannel().getId();
-        String[] itemGenChannelIds = NerdBotApp.getBot().getConfig().getItemGenChannel();
+    private MinecraftHead buildHead(GuildSlashEvent event, String textureID, Boolean isPlayerName) {
+        if (isPlayerName != null && isPlayerName) {
+            textureID = getPlayerHeadURL(event, textureID);
+            if (textureID == null) {
+                return null;
+            }
+        }
 
-        if (itemGenChannelIds == null) {
-            event.reply("The config for the item generating channel is not ready yet. Try again later!").setEphemeral(true).queue();
-            return;
+        if (textureID.contains("http://textures.minecraft.net/texture/")) {
+            textureID = textureID.replace("http://textures.minecraft.net/texture/", "");
+            event.getHook().sendMessage("Hey, a small heads up, but you only need to include the ID of the skin\n (not the `http://textures.minecraft.net/texture/` part")
```

Comment:
Should be reworded to sound better
eg. "Hey, a small heads up - you don't need to include the full URL! Only the skin ID is required"